CI/docker/build_moab.sh

```diff
@@ -14,19 +14,28 @@ rm -rf ${moab_build_dir}/bld ${moab_install_dir}
 mkdir -p ${moab_build_dir}/bld
 cd ${moab_build_dir}
 git clone --depth 1 https://bitbucket.org/fathomteam/moab -b ${branch}
-cd moab
-autoreconf -fi
-cd ../bld
-../moab/configure --enable-pymoab \
-                 --enable-shared \
-                 --enable-optimize \
-                 --disable-debug \
-                 --disable-fortran \
-                 --disable-blaslapack \
-                 --with-hdf5=${hdf5_install_dir} \
-                 --prefix=${moab_install_dir} \
-                 CC=${CC} CXX=${CXX}
-make -j${jobs}
+cd bld
+cmake ../moab -DBUILD_SHARED_LIBS=OFF \
+              -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_install_dir} \
+              -DENABLE_BLASLAPACK=OFF \
+              -DENABLE_FORTRAN=OFF \
+              -DCMAKE_INSTALL_PREFIX=${moab_install_dir} \
+              -DCMAKE_C_COMPILER=${CC} \
+              -DCMAKE_CXX_COMPILER=${CXX} \
+              -DCMAKE_INSTALL_RPATH=${hdf5_install_dir}:${moab_install_dir}
```

Comment:
I think that replacing
```
-DCMAKE_INSTALL_RPATH=${hdf5_install_dir}:${moab_install_dir}
```
with
```
-DCMAKE_INSTALL_RPATH=${hdf5_install_dir}/lib:${moab_install_dir}/lib
```
here will fix the current CI issue.

Reply:
https://github.com/svalinn/DAGMC/pull/687#issuecomment-656489283

Reply:
It looks like I might have dropped the `/lib` when adapting this in  the last update